src/pages/learn/index.tsx

```diff
@@ -0,0 +1,729 @@
+                  </Center>
+                  <ButtonLink to="/eth">What is ETH?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Where to get ETH?"
+                description="There are many ways how to get ETH depending on your
+              location."
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.impact)} />
+                  </Center>
+                  <ButtonLink to="/">Where to get ETH?</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <AdditionalReadingHeader>
+              Additional reading on Ethereum
+            </AdditionalReadingHeader>
+            <DocsContainer>
+              <DocLink to="/smart-contracts">What are Smart Contracts?</DocLink>
+              <DocLink to="/">A technical introduction to Ethereum</DocLink>
+              <DocLink to="/">What is Web3?</DocLink>
+              <DocLink to="/">Decentralizing Everything</DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[1].id}>{tocItems[1].title}</h2>
+            <p>
+              It’s actually quite simple once you get the hang of it. You need
+              an application that is commonly called a “wallet”. Its an app that
+              helps you store you funds and to authenticate/interact with
+              applications on Ethereum.
+            </p>
+            <CardGrid>
+              <Card
+                title="What is a wallet?"
+                description="How you manage your ETH and your Ethereum account."
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.wallet)} />
+                  </Center>
+                  <ButtonLink to="/wallets">What is a wallet?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="List of wallets"
+                description="We will help you find the right wallet for you."
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.futureTransparent)} />
+                  </Center>
+                  <ButtonLink to="/wallets">List of wallets</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Crypto security 101"
+                description="Learn how to identify scams and see the list of most common tricks."
+              >
+                <>
+                  <Center>
+                    <GatsbyImage image={getImage(data.dogeComputer)} />
+                  </Center>
+                  <ButtonLink to="/security">Crypto security 101</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <Banner>
+              <BannerBody>
+                <h3>Things to consider</h3>
```

Comment:
@konopkja slight preference for being more explicit here. headings and their content should mostly make sense in isolation from our context.

```suggestion
                <h3>Things to consider when using Ethereum</h3>
```